core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -96,18 +101,21 @@ export class Question {
     this._inapplicableSkillMisconceptionIds = (
       inapplicableSkillMisconceptionIds);
   }
-
-  getValidationErrorMessage(): string {
+  // Returns 'null' when the message is valid.
+  getValidationErrorMessage(): string | null {
     var interaction = this._stateData.interaction;
+    var interactionId = interaction.id as InteractionSpecsKey;
     var questionContent = this._stateData.content._html;
     if (questionContent.length === 0) {
       return 'Please enter a question.';
     }
     if (interaction.id === null) {
       return 'An interaction must be specified';
     }
-
-    if (interaction.defaultOutcome.feedback._html.length === 0) {
+    if (
+      interaction.defaultOutcome !== null &&
+      interaction.defaultOutcome.feedback._html.length === 0
+    ) {
```

Comment:
We can shorten this statement:
```suggestion
    if (
      interaction.defaultOutcome?.feedback._html.length === 0
    ) {
```

Reply:
Done